shared/src/simulation/events/transfer-specific-vehicle-request.ts

```diff
@@ -21,6 +26,10 @@ export class TransferSpecificVehicleRequestEvent implements SimulationEvent {
     @IsUUID(4, uuidValidationOptions)
     readonly transferDestinationId: UUID;
 
+    @IsOptional()
+    @Type(...occupationTypeOptions)
+    readonly successorOccupation?: ExerciseOccupation;
```

Comment:
This property is currently not in use. Could you please handle it in the transfer behavior?